recipes/pyhecdss/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pyhecdss" %}
+{% set version = "0.2.4" %}
+
+package:
+  name:           "{{ name|lower }}"
+  version:        "{{ version }}"
+
+source:
+  #url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/CADWRDeltaModeling/pyhecdss/tarball/{{version}}
+
+build:
+  number: 0
+  #skip:           true
+  skip: true # [osx]
+  skip: true # [py<35]
+  skip: true # [win and vc<14]
+  script: LDSHARED=g++ python setup.py install --single-version-externally-managed --record=record.txt # [unix]
```

Comment:
If you need gcc, please depend on GCC, rather than compilers("cxx")

Reply:
Can i list g++ as the dependency instead. See my build line for unix

Reply:
That is generally not advised.